application/views/admin/globalsettings/_email.php

```diff
@@ -93,6 +93,13 @@
                 <input class="form-control"  type='text' size='5' id='maxemails' name='maxemails' value="<?php echo htmlspecialchars(getGlobalSetting('maxemails')); ?>" />
             </div>
         </div>
+        <div class="form-group">
+            <label class="control-label" for="sendingrate"><?php eT("Email sending rate:"); ?></label>
+            <div>
+                <?php echo Chtml::textField("sendingrate", getGlobalSetting('maxemails'), array('class' => 'form-control', 'size' => '5')); ?>
```

Comment:
numberField maybe ? 

Else : https://github.com/LimeSurvey/LimeSurvey/blob/11d333b86731acb9be0b53d9f7b629a36a65c7f8/application/helpers/globalsettings_helper.php#L20 :)

Reply:
I tried to make a patch to be compatible with LS3.x branch as close as possible. I cannot find information when App()->getConfig($settingname)  was deprecated. Can I use it in LS3?

Reply:
I should I use App()->getConfig() or Yii::app->getConfig() in templates?

Reply:
This is not really fixed but some of dev prefer to use `App()`

Reply:
> I tried to make a patch to be compatible with LS3.x branch as close as possible. I cannot find information when getGlobalSettings() was deprecated. Can I use App()->getConfig() in LS3?

Yes :) https://github.com/LimeSurvey/LimeSurvey/blob/e8237925b5915b739c4049183d171b714b308585/application/helpers/globalsettings_helper.php#L58

Reply:
Done.